README.md

```diff
@@ -99,6 +99,29 @@ minikube tunnel
 
 ## Deploying Kardinal Manager to local cluster
 
+Configure it by setting the following environment variables:
+
+```bash
+KARDINAL_MANAGER_CLUSTER_CONFIG_ENDPOINT=http://localhost:8080/cluster-resources
+KARDINAL_MANAGER_FETCHER_JOB_DURATION_SECONDS=10
+KARDINAL_MANAGER_TENANT_UUID=36e22127-3c9e-4110-aa83-af552cd94b88
+```
+
+or in the `kardinal-manager/deployment/k8s.yaml`:
+
+```yaml
+  env:
+    - name: KARDINAL_MANAGER_CLUSTER_CONFIG_ENDPOINT
+      # This is valid for reaching out the Kardinal Kontrol if this is running on the host
+     value: "http://host.minikube.internal:8080/cluster-resources"
+    - name: KARDINAL_MANAGER_FETCHER_JOB_DURATION_SECONDS
+    value: "10"
+    - name: KARDINAL_MANAGER_TENANT_UUID
+    value: "36e22127-3c9e-4110-aa83-af552cd94b88"
+```
+
+NOTE: you can get your tenant UUID by running any CLI command
```

Comment:
users won't have to set the KARDINAL_MANAGER_TENANT_UUID env var by themself when `deploy manager` is implemented, which is my next task.